packages/ui/src/components/GroupOptionsSheet.tsx

```diff
@@ -74,13 +74,29 @@ export function ChatOptionsSheet({
     [currentUser, groupData?.members]
   );
 
-  const adminActions = useMemo(
-    () => [
+  const actions = useMemo(() => {
+    const actions = [];
+    actions.push(
       {
+        title: 'Copy group reference',
+        action: () => {},
+        icon: 'ArrowRef',
+      },
+      {
+        title: isPinned ? 'Unpin' : 'Pin',
+        action: () => {},
+        icon: 'Pin',
```

Comment:
Wanna hook this up with store.pinItem and store.unpinItem?

Reply:
Good call, wired up pin and copy ref